try-it-out/docs/pages/create-repository-for-pipeline-config.rst

```diff
@@ -0,0 +1,132 @@
+.. _Create Git Repository for a Pipeline Configuration:
```

Comment:
I like the idea of "try it out" just putting a ``pipeline_config.groovy`` in the sample app repo. 

it would be good to have a page that walks through what is it and what it does though.